### ISSUE TITLE
Force Auto Fullscreen is broken since 0.24.0

### ISSUE THREAD

**Issue by RAPXT** [polymorphicshade/NewPipe#254]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/polymorphicshade/NewPipe/releases/latest).
- [X] I am *not* able to reproduce the bug with the *same version* of [upstream NewPipe](https://github.com/TeamNewPipe/NewPipe/releases/).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/polymorphicshade/NewPipe/issues) or [closed](https://github.com/polymorphicshade/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/polymorphicshade/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.24.0

### Steps to reproduce the bug

Tap on Hamburger menu -> Settings -> Extras -> Enable "Force Auto Fullscreen"

Select a video and press play, the video should now start in fullscreen. But instead the video starts in a mini player.

I have also tried it with version 0.23.3 and it was working there, so it's broken since 0.24.0.

Thank you for your time & dedication ❤️

**Comment by Atemu** [comment]
How does upstream NewPipe behave?

**Comment by RAPXT** [comment]
Upstream NewPipe doesn't have this exact feature, which I described above, where it can be found.

But it has a similar feature, which can be found under:
Hamburger menu -> Settings -> Video and audio -> Behavior -> "Start main player in full screen"

I've enabled it and restarted the app. On version 0.24.0 it doesn't work (the video starts in mini-player instead of fullscreen). On version 0.23.3 it's also broken.

Summary: On upstream NewPipe it's at least broken since the last two versions.

On NewPipe SponsorBlock it was working till version 0.23.3 and is broken since version 0.24.0.
